src/main/java/com/karankumar/bookproject/ui/settings/SettingsView.java

```diff
@@ -47,10 +49,16 @@
 public class SettingsView extends HorizontalLayout {
 
     private static final String ENABLE_DARK_MODE = "Enable dark mode";
+    private static final String APPEARANCE = "Appearance:";
+    private static final String MYBOOKS = "My books:";
```

Comment:
For readability, could you add an underscore to separate "MY" from "BOOKS"?

Reply:
Done